paasta_tools/kubernetes_tools.py

```diff
@@ -3432,3 +3443,23 @@ def update_crds(
             success = False
 
     return success
+
+
+def get_kubernetes_secret_name(
+    service_name: str, secret_name: str, namespace: str = "paasta"
+) -> str:
+    service = sanitise_kubernetes_name(service_name)
+    sanitised_secret = sanitise_kubernetes_name(secret_name)
+    name = f"{namespace}-secret-{service}-{sanitised_secret}"
+    return name
+
+
+def get_kubernetes_secret(secret_name: str, service_name: str, cluster: str) -> str:
+    k8s_secret_name = get_kubernetes_secret_name(service_name, secret_name)
+
+    kube_client = KubeClient(config_file=KUBE_CONFIG_USER_PATH, context=cluster)
```

Comment:
@iiliescu2 we probably want to refactor this a little further to look like the other helpers in this file: we should probably do something like [create_secret](https://sourcegraph.yelpcorp.com/Yelp/paasta@badf618668149abff341bc8d2383ff4f9625de42/-/blob/paasta_tools/kubernetes_tools.py?L3012) and pass in a KubeClient - as it is, this helper cannot be used outside of cli tooling since it specifies a default kubeconfig file to use.

passing in a kubeclient would also make it so that we don't need to take in a cluster param here (which would be more consistent with the rest of the helpers in this file)

Reply:
Missed it then, but I included the refactoring in: https://github.com/Yelp/paasta/pull/3482